src/structured-data/structured-data-tags.js

```diff
@@ -218,9 +218,15 @@ function generateBreadcrumbListData(pageType = "", publisherConfig = {}, data =
     return sectionCrumbsDataList;
   }
 
+  function getMagazinePageCrumbs(magazine = {}) {
+    const crumbsDataList = [{ url: magazine.pagePath, name: magazine.name}];
+    return addCrumb(crumbsDataList, magazine);
+  }
+
   switch (pageType) {
     case "section-page": breadcrumbsDataList = breadcrumbsDataList.concat(getSectionPageCrumbs(data.section)); break;
     case "story-page": breadcrumbsDataList = breadcrumbsDataList.concat(getStoryPageCrumbs(data.story)); break;
+    case "magazine-page": breadcrumbsDataList = breadcrumbsDataList.concat(getMagazinePageCrumbs(data.collection)); break;
```

Comment:
@anbusurendhar-vikatan All Quintype libraries do not have context of a magazine. If you check #177, it is better to implement something more generic like breadcrumbs so which is not relevant to page type. This can be used for all custom pages in the future. In your case the magazine-page is a custom page. We will definitely relook at how section-page and story-page also fit into the same framework.